configurations/run_db_node_benchmarks.yaml

```diff
@@ -0,0 +1 @@
+run_db_node_benchmarks: true
```

Comment:
You don't use it anywhere.
Please add it to the Jenkins pipeline of the jobs we want to enable.

Reply:
i added to new jenkins file with enabled test for hw benchmark